### PR TITLE
refactor(ci): refactor helm GHA workflow

### PR DESCRIPTION
**Pull request by kaanyagci**
## Description

Closes FL-1240

### Added

- Checking if `Chart.lock` file is also updated

### Changed

- Existing helm GitHub Action workflow refactored to use `substra/substra-gha-workflows/.github/workflows/helm.yml`.

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -17,112 +17,7 @@ concurrency:
...
-          mv substra-backend-$(grep -e "^version" charts/substra-backend/Chart.yaml | cut -c10-).tgz substra-charts/
-          cd substra-charts
-          helm repo index .
-          git add .
-          git config --global user.email "gh-actions@github.com"
-          git config --global user.name "GitHub Action"
-          git commit -s --message "GitHub Action: ${{ github.repository }}@${{ github.sha }}"
-          git push --quiet --set-upstream origin main
+  test-generate-publish:
+      uses: substra/substra-gha-workflows/.github/workflows/helm.yml@chore/check-chart-lock-changes
```
Comment (oleobal) [inline .github/workflows/helm.yml:21]: Is it normal we are targeting the `chore/check-chart-lock-changes` rather than a long-lived branch or tag?

Comment (kaanyagci) [reply .github/workflows/helm.yml]: You're right! Just changed it

**Comment by kaanyagci** [comment]
[e56848c](https://github.com/Substra/substra-backend/pull/771/commits/e56848cded8b1148057456df942af8672e901170) refactors the `conventional_pr` GHA workflow to `substra-gha-workflows` repository. This changes the `conventional_title` workflow to `Conventional PR title / conventional_title / conventional_title`. As the `conventional_title` workflow is set as required, it hangs as it no longer exists. We need to change the required workflow or drop that commit.

**Comment by oleobal** [comment]
Tagging @Substra/code-owners since otherwise they won't get notified ^^

**Review by SdgJlbl — APPROVED** [review]
Thanks for your work 🙏

**Pull request merged**